src/builder/modules/Generator.res

```diff
@@ -156,6 +226,53 @@ let drawFoldLine = (from: Builder.position, to: Builder.position) => {
     )
 }
 
+let drawFoldLinePath = (points: array<Builder.position>, ~close: bool=false, ()) => {
+  model :=
+    Generator_Builder.drawPath(
+      model.contents,
+      points,
+      ~color="#7b7b7b",
+      ~width=1,
+      ~pattern=[2, 2],
+      ~offset=3,
+      ~close,
+    )
+}
+
+let drawFoldLineRect = (dest: Builder.rectangle) => {
```

Comment:
I believe your goal is to create a great experience by providing many choices to choose from - and thanks for these creative suggestions.

The consequence is the builder becomes significantly less simple when there are multiple versions of essentially the same thing. Unless there is a genuinely compelling reason to keep a function, let's omit it.